bigip/resource_bigip_ltm_virtual_server.go

```diff
@@ -401,7 +399,9 @@ func resourceBigipLtmVirtualServerUpdate(d *schema.ResourceData, meta interface{
 		TranslateAddress: d.Get("translate_address").(string),
 		VlansEnabled:     d.Get("vlans_enabled").(bool),
 	}
-
+	if d.Get("description").(string) != "" {
```

Comment:
same here, this conditional is not needed

Reply:
@dannyk81 
Updated with comments and submitted change